### DISCUSSION TITLE
How to download Beekeeper Community Edition?

### DISCUSSION THREAD

**Discussion by premgthb** [beekeeper-studio/beekeeper-studio#1364] [General]
The website only had links to download the ultimate edition, is there any guide to download the community edition? Thanks in advance.

**Comment by JonDeguz** [comment]
Second this^ there seems to be no way to download the community edition anymore.

**Reply by premgthb** [reply]
So after a bit of searching, I found out you can download the community edition by going to the Releases in the beekeeper-studio repository and download your required installer.

**Comment by rathboma** [comment]
Hey! Sorry for the confusion. I'm trying a bunch of experiments on the website and it's def not as clear as it should be where to download the community edition.

For future reference, go to this url to get the latest release: https://github.com/beekeeper-studio/beekeeper-studio/releases/latest

**Reply by KamilaMattos** [reply]
Hello! Sorry I'm a little confused, which version for windows should I download? Thanks!

**Reply by enshikuku** [reply]
Found it... Worked on win 11
https://github.com/beekeeper-studio/beekeeper-studio/releases/download/v3.7.9/Beekeeper-Studio-3.7.9-portable.exe

**Comment by enshikuku** [comment]
Hello I'm also a bit confused.. for the community edition which version for windows can I download

**Reply by enshikuku** [reply]
Found it... Worked on win 11
https://github.com/beekeeper-studio/beekeeper-studio/releases/download/v3.7.9/Beekeeper-Studio-3.7.9-portable.exe

**Reply by ysk3a** [reply]
Hi, I know this is really old post but I wanted to ask why you chose portable instead of setup version for windows for community version? And if one is better than the other what that reason is?

**Comment by rajlath** [comment]
Tried portable version from github , 
It does not have auto complete feature - is it available only on Paid version.
It does highlight key words though.

**Comment by ufukeskici** [comment]
Which file should I download from the following link to run community edition for MacOS M1?

[https://github.com/beekeeper-studio/beekeeper-studio/releases/tag/v3.8.9](https://github.com/beekeeper-studio/beekeeper-studio/releases/tag/v3.8.9)

**Reply by GraemeTG** [reply]
I'm not sure which version of community edition to download either for macOS Monterey 12.2.1 from this page https://github.com/beekeeper-studio/beekeeper-studio/releases/tag/v3.9.9 
Thank you

**Reply by GraemeTG** [reply]
I've found the one - for me it is Beekeeper-Studio-3.9.9.dmg
